config/staging.json

```diff
@@ -0,0 +1,19 @@
+{
+  "logger": {
+    "level": "debug",
+    "fileName": "logs/staging.log",
+    "maxSize": 1024000,
+    "maxFiles": 4,
+    "logzio": {
+      "enabled": false
+    }
+  },
+  "consensus": {
+    "clusterSize": 7,
```

Comment:
@netoneko just to double check - are we running 7 nodes on staging indeed? Not 6, like via `simulate.sh`?

Reply:
Yes, we are running 7 nodes.